bbpower/compsep.py

```diff
@@ -346,7 +350,17 @@ def model(self, params):
             bmat = np.array([[c, s],
                              [-s, c]])
             cmb_cell = rotate_cells_mat(bmat, bmat, cmb_cell)
-        
+
+        self.analytic_TempxTemp = np.zeros([self.n_ell, self.npol, self.npol])
+        temp_file = np.loadtxt(self.get_input('analytic_temp_BB'))
```

Comment:
we don't want to read the template at every point in the likelihood. Do all of this at the initialization stage